pygeoapi/flask_app.py

```diff
@@ -144,6 +181,13 @@ def describe_processes(name=None):
 
 @APP.route('/processes/<name>/jobs', methods=['GET', 'POST'])
 def execute_process(name=None):
+    """
+    OGC open api jobs from processes access point (experimental)
+
+    :param name: identifier of process to execute
+    :returns: HTTP response
```

Comment:
should be `:returns: `dict` of response, or something similar.  In other words, provide the Python type of the response (should be done for all functions).

Reply:
Can we keep it as it is, since I am not certain of what is the nature of the flask HTTP responde ?